src/Business Foundation/App/NoSeries/src/Setup/NoSeries.Page.al

```diff
@@ -0,0 +1,302 @@
+                    Editable = false;
+                    ToolTip = 'Specifies the value for incrementing the numeric part of the series.';
+                    Visible = false;
+
+                    trigger OnDrillDown()
+                    var
+                        NoSeriesMgt: Codeunit NoSeriesMgt;
+                    begin
+                        NoSeriesMgt.DrillDown(Rec);
+                        CurrPage.Update(false);
+                    end;
+                }
+                field("Default Nos."; Rec."Default Nos.")
+                {
+                    ApplicationArea = Basic, Suite;
+                    ToolTip = 'Specifies whether this number series will be used to assign numbers automatically.';
+                }
+                field("Manual Nos."; Rec."Manual Nos.")
+                {
+                    ApplicationArea = Basic, Suite;
+                    ToolTip = 'Specifies that you can enter numbers manually instead of using this number series.';
+                }
+                field("Date Order"; Rec."Date Order")
+                {
+                    ApplicationArea = Basic, Suite;
+                    ToolTip = 'Specifies to check that numbers are assigned chronologically.';
+                }
+                field(AllowGapsCtrl; AllowGaps)
+                {
+                    ApplicationArea = Basic, Suite;
+                    Caption = 'Allow Gaps in Nos.';
+                    ToolTip = 'Specifies that a number assigned from the number series can later be deleted. This is practical for records, such as item cards and warehouse documents that, unlike financial transactions, can be deleted and cause gaps in the number sequence. This setting also means that new numbers will be generated and assigned in a faster, non-blocking way. NOTE: If an error occurs on a new record that will be assigned a number from such a number series when it is completed, the number in question will be lost, causing a gap in the sequence.';
+
+                    trigger OnValidate()
+                    var
+                        NoSeriesMgt: Codeunit NoSeriesMgt;
+                    begin
+                        Rec.TestField(Code);
+                        NoSeriesMgt.SetAllowGaps(Rec, AllowGaps);
+                    end;
+                }
+            }
+        }
+        area(factboxes)
+        {
+            systempart(Control1900383207; Links)
+            {
+                ApplicationArea = RecordLinks;
+                Visible = false;
+            }
+            systempart(Control1905767507; Notes)
+            {
+                ApplicationArea = Notes;
+                Visible = false;
+            }
+        }
+    }
+
+#pragma warning disable AA0219 // The Tooltip property for PageAction must start with 'Specifies'.
+    actions
+    {
+        area(navigation)
+        {
+            group("&Series")
+            {
+                Caption = '&Series';
```

Comment:
Does it still make sense include '&' in action captions?

Reply:
100% valid point, h/e the pr desc. says "in a non-breaking solution", and as you've mentioned in one of your above comments, it's focus is on interfaces